src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/nomis/UserApiService.kt

```diff
@@ -111,6 +112,14 @@ class UserApiService(
     return serviceWebClientUtils.get("/users/{username}", PrisonUser::class.java, username.uppercase())
   }
 
+  fun findUserDetailsByUsername(username: String): PrisonUserDetails? {
```

Comment:
This is a good check (and as far as I know is correct) but get that verified from Adrian .
Can we add this at UI?

Reply:
To be honest I was quite surprised to see it but that check is made in all the parallel `getUser` type functions in `/nomis/UserApiService.kt` - `findUserDetailsByUsername` , `findUserByUsernameWithError`, `findUserBasicDetailsByUsername` and `findUserByUsername`
I think it is okay for this PR